src/login/login.php

```diff
@@ -1,18 +1,26 @@
 <?php
     session_start();
 
+    require_once( realpath( dirname( __FILE__ ) ) . "/../php/DAL.php" );
+
+    $dal = new DAL();
+    $user_id = $dal->query("SELECT user_id FROM User WHERE username='$username', password='$password'");
     $username = $_POST['user1'];
     $password = $_POST['password1'];
 
     $_SESSION[ "user_id" ] = "3";
-    $_SESSION[ "username" ] = $username;
-    $_SESSION[ "password" ] = $password;
```

Comment:
Not sure if session variables are still needed for `username` and `password` (commented out lines 8 and 9)  since the sql is now being queried for these values on line 7 and set to`user_id`...